packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -717,9 +717,9 @@ class HotRunner extends ResidentRunner {
     const String bold = '\u001B[0;1m';
     const String reset = '\u001B[0m';
     printStatus(
-      '$fire  To hot reload your app on the fly, press "r". To restart the app entirely, press "R".',
+      '$fire  To hot reload your app on the fly, press "r". To hot restart the app, press "R".',
```

Comment:
Given how close the names are, is it worth trying to differentiate what the two actions do here? 

`To hot reload changes into your app, press "r". To hot restart the app, press "R".`

Reply:
How about this?

`  To hot reload changes while running, press "r". To hot restart and rebuild state, press "R".`

Reply:
`To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".`?